inst/examples/germany_reporting_cycle_model.R

```diff
@@ -0,0 +1,124 @@
+# Load packages
+library(epinowcast)
+library(data.table)
+
+# Set cmdstan path
+cmdstanr::set_cmdstan_path()
+
+# Use 2 cores
+options(mc.cores = 2)
+
+# Load and filter germany hospitalisations
+nat_germany_hosp <- germany_covid19_hosp[location == "DE"][age_group == "00+"]
+nat_germany_hosp <- enw_filter_report_dates(
+  nat_germany_hosp,
+  latest_date = "2021-10-01"
+)
+
+nat_germany_hosp <- enw_filter_reference_dates(
+  nat_germany_hosp,
+  earliest_date = "2021-07-01"
+)
+
+nat_germany_hosp <- enw_complete_dates(
+  nat_germany_hosp,
+  by = c("location", "age_group"),
+  timestep = "day"
+)
+
+# Aggregate data to weekly reporting cycle
+repcycle_germany_hosp <- nat_germany_hosp |>
+  enw_add_incidence() |>
+  dplyr::mutate(day_of_week = lubridate::wday(report_date, label = TRUE)) |>
+  dplyr::mutate(confirm = new_confirm) |>
+  # Aggregate rolling sum sums over "confirm"
+  # but we want a sum over "new_confirm"
+  epinowcast:::aggregate_rolling_sum(
```

Comment:
this is starting to be used a bit in places do you think we should export it?

Reply:
No strong opinions on this, I think I've only used it to create reporting cycle style data, which I don't think is that common to need to do